File: ios/RNCharger.swift
//
//  RNCharger.swift
//  HeychargeSdk
//
//  Created by Muhammad Mneimneh on 15.02.2023.
//  Copyright © 2023 Facebook. All rights reserved.
//

import Foundation
import ios_sdk

struct RNCharger: Codable {
    let charger: Charger
    let properties: RNChargerProperties
}

struct RNChargerProperties: Codable {
    let isAvailable: Bool
    let isChargingByUser: Bool
    let isChargingBusy: Bool
    let isChargerUpdateAvailable: Bool
}

File: android/src/main/java/com/heychargesdk/RNCharger.kt
package com.heychargesdk

import com.heycharge.androidsdk.domain.Charger


data class RNCharger(val charger: Charger, val properties: RNChargerProperties)

data class RNChargerProperties(
  val isAvailable: Boolean,
  val isChargingByUser: Boolean,
  val isChargingBusy: Boolean,
  val isChargerUpdateAvailable: Boolean
)
